src/runtime/TemporalObject.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include "Escargot.h"
 #include "TemporalObject.h"
+#include <iomanip>
```

Comment:
Is this standard c++?

Reply:
Yes it is.

Reply:
Where is this lib used for?
Is this mandatory?

Reply:
In line 1550. If you can suggest me an easier way to write that piece of code then let me know.